modmesh/app/sample_mesh.py

```diff
@@ -222,6 +236,112 @@ def make_3dmix():
     return mh
 
 
+def make_solvcon():
+    Q = core.StaticMesh.QUADRILATERAL
+    mh = core.StaticMesh(ndim=2, nnode=140, nface=0, ncell=65)
+    mh.ndcrd.ndarray[:, :] = [
+            (0, 0), (1, 0), (2, 0), (3, 0), (4, 0), (5, 0), (6, 0),
+            (7, 0), (8, 0), (9, 0), (10, 0), (11, 0), (12, 0), (13, 0),
+            (14, 0), (15, 0), (16, 0), (18, 0), (19, 0), (20, 0), (21, 0),
+            (22, 0), (23, 0), (24, 0), (25, 0), (26, 0), (27, 0), (28, 0),
+            (29, 0), (0, 1), (1, 1), (2, 1), (3, 1), (4, 1), (5, 1), (6, 1),
+            (7, 1), (8, 1), (9, 1), (10, 1), (11, 1), (12, 1), (13, 1),
+            (14, 1), (15, 1), (16, 1), (18, 1), (19, 1), (20, 1), (21, 1),
+            (22, 1), (23, 1), (24, 1), (25, 1), (26, 1), (27, 1), (28, 1),
+            (29, 1), (0, 2), (1, 2), (2, 2), (3, 2), (4, 2), (5, 2), (6, 2),
+            (7, 2), (8, 2), (9, 2), (12, 2), (13, 2), (15, 2), (16, 2),
+            (18, 2), (19, 2), (22, 2), (23, 2), (24, 2), (25, 2), (26, 2),
+            (27, 2), (28, 2), (29, 2), (0, 3), (1, 3), (2, 3), (3, 3), (4, 3),
+            (5, 3), (6, 3), (7, 3), (8, 3), (9, 3), (11, 3), (12, 3), (13, 3),
+            (15, 3), (16, 3), (17, 3), (18, 3), (19, 3), (20, 3), (21, 3),
+            (22, 3), (23, 3), (24, 3), (25, 3), (26, 3), (27, 3), (28, 3),
+            (29, 3), (0, 4), (1, 4), (2, 4), (3, 4), (4, 4), (5, 4), (6, 4),
+            (7, 4), (8, 4), (9, 4), (11, 4), (12, 4), (13, 4), (15, 4),
+            (16, 4), (17, 4), (18, 4), (19, 4), (20, 4), (21, 4), (22, 4),
+            (23, 4), (24, 4), (25, 4), (26, 4), (27, 4), (0, 5), (1, 5),
+            (2, 5), (3, 5)
+        ]
+    mh.cltpn.ndarray[:] = [
+           Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,
+           Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,  # 0-20
+           Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,  # 21-31
+           Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,  # 32-45
+           Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q, Q,  # 46-61
+           Q, Q, Q  # 62-64
+        ]
+    mh.clnds.ndarray[:, :5] = [
+            (4, 0, 1, 30, 29),
```

Comment:
@yungyuc  isn't it better to put this in a data file rather than hard-coded in a py file?